docs/docs/cli.md

```diff
@@ -26,6 +28,14 @@ Unless you've installed `sku` globally (not recommended), you should use `npx` o
 pnpm dlx sku init my-app
 ```
 
+This command supports the following options and arguments:
+
+| Type     | Name                           | Description                                                                                                   |
+| -------- | ------------------------------ | ------------------------------------------------------------------------------------------------------------- |
+| argument | `project-name` <br> _required_ | The name of the project to create <br> `pnpm dlx sku init my-app`                                             |
+| option   | `--package-manager, -p`        | The package manager to use when installing dependencies <br> `pnpm dlx sku init my-app --package-manager=npm` |
```

Comment:
Worth mentioning that this defaults to the package manager being used to run `sku init`, so it's really more of an override.